client/ayon_houdini/api/pipeline.py

```diff
@@ -438,18 +448,28 @@ def _enforce_start_frame():
         _enforce_start_frame()
 
 
-def get_or_create_avalon_container() -> "hou.OpNode":
-    avalon_container = hou.node(AVALON_CONTAINERS)
-    if avalon_container:
-        return avalon_container
+def get_or_create_ayon_container() -> "hou.OpNode":
+    ayon_container = hou.node(AYON_CONTAINERS)
+    if ayon_container:
+        return ayon_container
+
+    # Allow legacy name for the containers
+    legacy_ayon_container = hou.node(LEGACY_AVALON_CONTAINERS)
+    if legacy_ayon_container:
+        return legacy_ayon_container
 
-    parent_path, name = AVALON_CONTAINERS.rsplit("/", 1)
+    parent_path, name = AYON_CONTAINERS.rsplit("/", 1)
     parent = hou.node(parent_path)
     return parent.createNode(
         "subnet", node_name=name
     )
 
 
+def get_or_create_avalon_container():
+    """Deprecated, please use `get_or_create_ayon_container` instead."""
+    return get_or_create_ayon_container()
```

Comment:
@antirotor @iLLiCiTiT any pointers on marking this deprecated the best way, without needing to put in code like [this](https://github.com/ynput/ayon-nuke/blob/b055b641793036c13f5f3a9b12548b6fb050ecfc/client/ayon_nuke/api/lib.py#L83-L125) into this codebase just to mark deprecated?

Reply:
I usually use `warnings.warn("...what, why...", DeprecationError)` (`import warnings`).

And also add `deprecated` section in docstring so IDEs can mark it in code, but you already did that...